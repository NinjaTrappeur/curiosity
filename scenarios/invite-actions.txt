reset
user invite alice@example.com
queues
