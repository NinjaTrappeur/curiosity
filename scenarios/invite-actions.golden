1: reset
Resetting to the empty state.
2: user invite alice@example.com
User created: USER-1
3: queues
Email addresses to verify:
USER-1 /
Emails to send:
EMAIL-1 alice@example.com
